.travis/package_management/common.py

```diff
@@ -80,7 +80,11 @@ def prepare_repo(container):
             run_command(container, [os.environ["REPO_TOOL"], "install", "-y", "epel-release"])
 
     elif str(os.environ["REPO_TOOL"]).count("apt-get") == 1:
-        run_command(container, [os.environ["REPO_TOOL"], "update", "-y"])
+        if str(os.environ["BUILD_STRING"]).count("debian/jessie") == 1:
```

Comment:
This ought to be the weirdest alternative to the more robust `"debia/jessie" in os.getenv("BUILD_STRING", "")` !!! I see it's been previously used elsewhere, but why!!!!!

Reply:
I'm not sure myself, but I'm just assuming whoever wrote it knew what they were doing.

I'll probably update this to be more efficient (and saner) when I'm working on switching things over to Docker instead of LXC for the build, but for now it works and fixes an issue that's causing the build to fail.